supported-libraries-schema.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
As a user seeing `supported-libraries-schema.json`, I'd think this is about the libraries supported by this repo. Also, I think we should include frameworks as well. How about we rename this to `library-and-framework-support-schema.json` and the data is stored in `library-and-framework-support.json`?